Source/UINavigation/Public/UINavInputDisplay.h

```diff
@@ -37,6 +38,10 @@ class UINAVIGATION_API UUINavInputDisplay : public UUserWidget
 	UFUNCTION(BlueprintCallable, Category = "InputDisplay")
 	void SetInputAction(UInputAction* NewAction, const EInputAxis NewAxis, const EAxisType NewScale);
 
+	// Locks to a specific input type. If set to None, it is automatically detected
+	UPROPERTY(BlueprintReadWrite,EditAnywhere,meta=(ExposeOnSpawn),Category="InputDisplay")
```

Comment:
Add proper spacing here bettwen ",". Also why the expose on spawn?

Reply:
Thought I needed it for being able to edit in Class Defaults on a child, but I see now that EditAnywhere is enough